metagraph/src/graph/alignment/aligner_config.hpp

```diff
@@ -66,16 +68,27 @@ class DBGAlignerConfig {
     score_t min_path_score = 0;
     score_t xdrop = std::numeric_limits<score_t>::max();
 
+    DeBruijnGraph::node_index terminal_node = DeBruijnGraph::npos;
```

Comment:
So you keep this in the config, but not the sequence. This is weird.

Reply:
I would recommend to keep sequence here as well and not pass it as a separate input parameter, or pass this terminal node as a separate parameter, similarly to `query`

Reply:
I'll add a comment describing this. The point is that I want to define an endpoint for an alignment by setting a node and a target traversal distance at which to stop.

Reply:
I got that. The thing I don't get is why you want to keep this and the sequence itself separately.
I could understand if you keep alignment parameters in a struct separately from the objects (sequence + node) or just keep all these things in one struct and call it `AlingmentContext`, or something.

Reply:
I changed this so now target_distance and target_node are passed as arguments to extend.